src/UsblParser.cpp

```diff
@@ -410,8 +410,6 @@ DeviceSettings UsblParser::parseCurrentSettings (string const &buffer)
             settings.carrierWaveformId = atoi(splitted2.at(1).c_str());
         else if(splitted2.at(0).find("Local Address") != string::npos)
             settings.localAddress = atoi(splitted2.at(1).c_str());
-        else if(splitted2.at(0).find("Cluster Size") != string::npos)
-            settings.carrierWaveformId = atoi(splitted2.at(1).c_str());
```

Comment:
Few lines below is the right Cluster Size. It was wrongly placed here (a bad copy and paste)